src/patterns/data/manual/xpoAnatomyInLocationTEMP.tsv

```diff
@@ -332,3 +309,11 @@ http://purl.obolibrary.org/obo/XPO_0116165	PATO:0002481	oblique orientation	XAO:
 http://purl.obolibrary.org/obo/XPO_0116166	PATO:0005013	incomplete structure	XAO:0004013	XAO:0000099
 http://purl.obolibrary.org/obo/XPO_0116167	PATO:0005013	incomplete structure	XAO:0004013	XAO:0003024
 http://purl.obolibrary.org/obo/XPO_0116168	PATO:0005020	irregularly shaped	XAO:0003249	XAO:0004513
+http://purl.obolibrary.org/obo/XPO_0115898	PATO:0000001	quality	XAO:0003213	XAO:0004107
```

Comment:
What are all these addiotions here? I expected to see only removals :P

Reply:
These items were identified as duplicates but had incorrect part_of directionality; the items were removed, then the anatomical entity and location were swapped and the items added back at the end of the file.

Reply:
I would like to be able to match them back to the original annotations; can you maintain a list of all classes where directionality was changed? Perhaps in Gdocs?